posthog/temporal/data_imports/external_data_job.py

```diff
@@ -209,14 +157,6 @@ def parse_inputs(inputs: list[str]) -> ExternalDataWorkflowInputs:
     async def run(self, inputs: ExternalDataWorkflowInputs):
         assert inputs.external_data_schema_id is not None
 
-        if settings.TEMPORAL_TASK_QUEUE != DATA_WAREHOUSE_TASK_QUEUE_V2 and not TEST:
-            await workflow.execute_activity(
```

Comment:
As mentioned on standup, I believe with this, any workflow that gets resumed that has triggered this will end up in a bad state. (Should be minimal though so we can just reboot)